slave.mk

```diff
@@ -268,6 +268,11 @@ export SONIC_ROUTING_STACK
 export FRR_USER_UID
 export FRR_USER_GID
 
+###############################################################################
+## Build Options
+###############################################################################
+export DEB_BUILD_MAINT_OPTIONS = hardening=+all
```

Comment:
The manpage suggests that `DEB_BUILD_MAINT_OPTIONS` should be used only within `debian/rules`, and `DEB_BUILD_OPTIONS` should be used outside `debian/rules`, so that hardening options can be easily overridden in `debian/rules` if necessary. Given this, should this be changed to use `DEB_BUILD_OPTIONS` instead?

Reply:
@saiarcot895 , thanks for your comment. Changed to use DEB_BUILD_OPTIONS instead.